### PR TITLE
Updated: workflow

### PR DESCRIPTION
**Pull request by GoliathLabs**
## Describe your Pull Request
A clear and concise description of what the Pull Request is about.
Updated workflow and added additional architectures.

Fixes # (issue)

Please select relevant options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [X] My code follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have checked my code and corrected any misspellings

Reviewer: @StiviiK

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - id: read-docker-image-identifiers
+        name: Read Docker Image Identifiers
+        run: echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
+      - name: Build container image
```
Comment (StiviiK) [inline .github/workflows/docker.yaml:35]: Is there an specific reason why you don't use the cache in this step?

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - id: read-docker-image-identifiers
+        name: Read Docker Image Identifiers
+        run: echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
```
Comment (StiviiK) [inline .github/workflows/docker.yaml]: Why is this step required?

Comment (GoliathLabs) [reply .github/workflows/docker.yaml]: Isn't neccessary. I'm using it for ghcr.io so I'll remove it.

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (GoliathLabs) [reply .github/workflows/docker.yaml:35]: How can I use the cache in this step?

Comment (StiviiK) [reply .github/workflows/docker.yaml:35]: You used it in the integration step / action. Won't that work here the same?
```yaml
cache-from: type=gha
cache-to: type=gha,mode=max 
```

File: .github/workflows/docker.yaml (hunk 2 above)
Comment (StiviiK) [reply .github/workflows/docker.yaml]: @GoliathLabs any updates on this?

**Comment by GoliathLabs** [comment]
Sorry, I currently have no time on my hands :/

**Comment by GoliathLabs** [comment]
Hi, I've updated the PR :) Feel free to have a look

**Pull request closed without merging**